src/middlewared/middlewared/plugins/device_/device_info_freebsd.py

```diff
@@ -67,7 +67,7 @@ async def get_disk_details(self, disk, disk_geom):
                     disk['type'] = 'SSD'
                 else:
                     disk['type'] = 'HDD'
-            else:
+            elif disk_geom.provider.config.get('rotationrate') != 'unknown':
```

Comment:
What disk `type` is going to be in this case?

Reply:
It's going to be `UNKNOWN` ( we default to that )